test-clients/src/main/java/com/hedera/services/bdd/suites/crypto/MiscCryptoSuite.java

```diff
@@ -38,6 +39,7 @@
 import static com.hedera.services.bdd.spec.utilops.UtilVerbs.*;
 import static com.hedera.services.bdd.spec.queries.QueryVerbs.*;
 import static com.hederahashgraph.api.proto.java.ResponseCodeEnum.OK;
+import static com.hederahashgraph.api.proto.java.ResponseCodeEnum.SUCCESS;
 
 public class MiscCryptoSuite extends HapiApiSuite {
```

Comment:
I don't think this suite runs in CI; you may want to move your EET spec here to the `CryptoUpdateSuite`.